demos/TicTacToe/Classes/TicTacToeScene.cpp

```diff
@@ -0,0 +1,78 @@
+#include "TicTacToeScene.h"
+
+#include <array>
+#include <cstdlib>
+
+#include "cocos2d.h"
+
+USING_NS_CC;
+
+static const int kTilesX = 3;
+static const int kTilesY = 3;
+static const int kNumberOfTiles = kTilesX * kTilesY;
+static const int kMaxMovesPerPlayer = 1 + kNumberOfTiles / 2;
+static const double kScreenWidth = 600;
+static const double kScreenHeight = 600;
+static const double kTileWidth = (kScreenWidth / kTilesX);
+static const double kTileHeight = (kScreenHeight / kTilesY);
+static const int kNumberOfPlayers = 2;
+static const char* kBoardImageFileName = "tic_tac_toe_board.png";
+std::array<const char*, kNumberOfPlayers> kMoveImageFileNames = {
```

Comment:
Can we use something other than MoveImageFileNames?  The move term is confusing to me -- at first I thought we were moving the files names.  How about kPlayerTokenFileNames or kPlayerSymbolFileNames?